Community/ideAlarm/automation/lib/python/community/idealarm/__init__.py

```diff
@@ -1,17 +1,25 @@
 # -*- coding: utf-8 -*-
-
 import weakref # Using this to prevent problems with garbage collection
 
 from core.jsr223 import scope
 from core.log import logging, LOG_PREFIX
+
+import sys ############ TEMP TODO:
 from core.utils import isActive, getItemValue, postUpdateCheckFirst, sendCommandCheckFirst, kw
-from community.idealarm import custom
+reload(sys.modules['core.utils'])
+from core.utils import isActive, getItemValue, postUpdateCheckFirst, sendCommandCheckFirst, kw
+
+import sys # Trying to reload custom from personal.idealarm TODO: Does this actually work? 
+from personal.idealarm import custom
+reload(sys.modules['personal.idealarm'])
+from personal.idealarm import custom
 
 from org.eclipse.smarthome.model.persistence.extensions import PersistenceExtensions
```

Comment:
Let's use the actions module here... one less thing to update when the package names are changed in OH 3.0.

Reply:
Thanks. Fixed that.